views/result.erb

```diff
@@ -109,7 +109,7 @@
                   <div class="hit-links text-right">
                     <label>
                       <input
-                      type="checkbox" value=<%="#{ hit.accession }"%>>
+                      type="checkbox" value="<%="Query_#{query.number}_hit_#{hit.number}"%>">
```

Comment:
Depends on how one interprets the difference between `data-target` and
`value`. While former gives a sense that we are referring to a div or any
DOM element, latter doesn't.

So, if we are directly using `#Query_1_hit_1`, then might as well use
`data-target`.

On Wed Jan 28 2015 at 8:42:22 PM Anurag Priyam <notifications@github.com>
wrote:

> In views/result.erb
> <https://github.com/yeban/sequenceserver/pull/32#discussion_r23692852>:
>
> > @@ -109,7 +109,7 @@
> >                    <div class="hit-links text-right">
> >                      <label>
> >                        <input
> > -                      type="checkbox" value=<%="#{ hit.accession }"%>>
> > +                      type="checkbox" value="<%="Query_#{query.number}_hit_#{hit.number}"%>">
>
> Why not set the value directly to '#Query_1_hit_1'? Like BS does it for
> data-target?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/yeban/sequenceserver/pull/32/files#r23692852>.
>